app/models/access.rb

```diff
@@ -0,0 +1,38 @@
+class Access < ApplicationRecord
```

Comment:
Suggestion: rename this to `Role` or maybe `UserResource`, as it is a join model between User and Resource.

Reply:
It used to be called `UserResource`, but we found it very unwieldy to do `user.user_resources` and felt like `Role` doesn't really capture what this model does and neither does `UserResource` – but I'd agree that it is a little more explicit in the parlance of how-to-name-join-models.

Reply:
I'm not a fan of `UserResource`, but I think that's because I'm not a big fan of `Resource` in the first place. Also, there's enough behavior in here that I'd be ok deviating from the usual join-table naming in favor of something more semantic. Does this read better if it's an `Access` that has a `scope` instead of a `resource`? Resource seems very generic, whereas as scope is commonly used in the context of authorization.

Reply:
I'm thinking `region` is a name that makes more sense ...an Admin has access to a Region via a particular Role. 

I agree that Resource feels too generic right now...